samples/aspnetapp/Dockerfile.windowsservercore

```
@@ -1,6 +1,7 @@
 # Learn about building .NET container images:
 # https://github.com/dotnet/dotnet-docker/blob/main/samples/README.md
-FROM mcr.microsoft.com/dotnet/sdk:9.0-windowsservercore-ltsc2022 AS build
+ARG TAG=ltsc2022
```

The default should be the newest version, `ltsc2025`. This also applies to other default ARGs in sample Dockerfiles.

```suggestion
ARG TAG=ltsc2025
```